Okta.Xamarin/Okta.Xamarin/IOktaStateManager.cs

```diff
@@ -16,38 +16,125 @@ namespace Okta.Xamarin
     /// </summary>
     public interface IOktaStateManager
     {
+        /// <summary>
+        /// Gets the access token.
+        /// </summary>
         string AccessToken { get; }
 
+        /// <summary>
+        /// Gets or sets the configuration.
+        /// </summary>
         IOktaConfig Config { get; set; }
 
+        /// <summary>
+        /// Gets or sets the client.
+        /// </summary>
+        IOidcClient Client { get; set; }
+
+        /// <summary>
+        /// Gets the expiration time.
+        /// </summary>
         DateTime? Expires { get; }
```

Comment:
For expiration, we should use [`DateTimeOffset?`](https://docs.microsoft.com/en-us/dotnet/standard/datetime/choosing-between-datetime#the-datetimeoffset-structure) instead.